plugins/destination/duckdb/client/client_test.go

```diff
@@ -28,5 +33,15 @@ func TestPlugin(t *testing.T) {
 		destination.PluginTestSuiteTests{
 			MigrateStrategyOverwrite: migrateStrategy,
 			MigrateStrategyAppend:    migrateStrategy,
-		})
+			SkipOverwrite:            true,
+		},
+		// not supported in Parquet Writer
+		destination.WithTestSourceSkipIntervals(),
+		// not supported in Parquet Writer
```

Comment:
```suggestion
```

Reply:
why remove? those two types are not supported in parquet writer for now?

Reply:
it's not really obvious why we have two of the same comment in succession. you should also separate the other types (times and so on) with a newline as well.